src/stactools/noaa_sst/constants.py

```diff
@@ -0,0 +1,27 @@
+from datetime import datetime
+from pyproj import CRS
+from pystac import Provider, ProviderRole, Link
+
+NOAA_SST_ID = "noaa-sst"
+SST_EPSG = 4326
+SST_CRS = CRS.from_epsg(SST_EPSG)
+LICENSE = "proprietary"
+lic_link = "https://github.com/stactools-packages/noaa-sst/tree/main/src/stactools/noaa_sst/license"
+"/data-license.txt"
+LICENSE_LINK = Link(rel="license",
+                    target=lic_link,
+                    title="Public Domain License - NOAA")
+SPATIAL_EXTENT = [-180.0, 90.0, 180.0, -90.0]
```

Comment:
This is incorrect order for bbox ([W, S, E, N]): https://github.com/radiantearth/stac-spec/blob/master/item-spec/item-spec.md#bbox

Reply:
Fixed